dubbo-remoting/dubbo-remoting-api/src/test/java/org/apache/dubbo/remoting/exchange/support/DefaultFutureTest.java

```diff
@@ -116,6 +122,35 @@ public void timeoutSend() throws Exception {
         }
     }
 
+    @Test
+    public void interruptSend() throws Exception {
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        System.out.println("before a future is create , time is : " + LocalDateTime.now().format(formatter));
+        // timeout after 5 seconds.
+        Channel channel = new MockedChannel();
+        Request request = new Request(10);
+        ExecutorService sharedExecutor = ExtensionLoader.getExtensionLoader(ExecutorRepository.class)
+                .getDefaultExtension().createExecutorIfAbsent(URL.valueOf("dubbo://127.0.0.1:23456"));
+        ThreadlessExecutor executor = new ThreadlessExecutor(sharedExecutor);
+        DefaultFuture f = DefaultFuture.newFuture(channel, request, 5000, executor);
+        //mark the future is sent
+        DefaultFuture.sent(channel, request);
+        // get operate will throw a timeout exception, because the future is timeout.
```

Comment:
this comment should  change to this: get operate will throw a interrupted exception, because the thread is interrupted.